panther_docking/launch/docking.launch.py

```diff
@@ -36,11 +36,13 @@ def generate_launch_description():
         description=("Path to docking server configuration file."),
     )
 
-    declare_use_docking_arg = DeclareLaunchArgument(
-        "use_docking",
-        default_value="True",
-        description="Enable docking server.",
-        choices=["True", "False", "true", "false"],
+    apriltag_config_path = LaunchConfiguration("apriltag_config_path")
+    declare_apriltag_config_path_arg = DeclareLaunchArgument(
+        "apriltag_config_path",
+        default_value=PathJoinSubstitution(
+            [FindPackageShare("panther_docking"), "config", "apriltag.yaml"]
+        ),
+        description=("Path to apriltag configuration file. Only used in simulation."),
     )
 
     namespace = LaunchConfiguration("namespace")
```

Comment:
Please check if ros2 launch panther_docking docking.launch.py executs correctly if not this should fix this issue.

```suggestion
    namespace = LaunchConfiguration("namespace", default="")
    use_docking = LaunchConfiguration("use_docking", default="true")
    use_sim = LaunchConfiguration("use_sim", default="false")
```